app/[domain]/[lang]/[plan]/(with-layout-elements)/layout.tsx

```diff
@@ -20,6 +21,9 @@ type Props = {
 export default function Layout({ children, params }: Props) {
   const plan = usePlan();
   const { navigationTitle, title } = getMetaTitles(plan);
+  const session = useSession();
+
+  console.log('SESSION', session);
```

Comment:
@lilia1891 You'll be able to use this to check whether the user is authenticated, if `session.status === 'authenticated'` then we should show the new top bar.